### PR TITLE
Minor fixes and improvement: typos and simplifications

### PR DESCRIPTION
**Pull request by ryukinix**
+ Fix minor typos in model_report and project_specification
+ Simplify setup.py: avoid import model_name
+ Fix template/README.md instructions of git clone (use HTTPS as default instead SSH)

### CODE REVIEW

**Comment by ryukinix** [comment]
Can you review this @vhdeluca?

File: template/README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Make sure that **docker** is installed.
 
 Clone the project from the analytics Models repo.
 ```
-git clone git@github.com:NeowayLabs/data-science-template.git
+git clone https://github.com/<@github_username>/<@project>.git
```
Comment (ryukinix) [inline template/README.md:30]: Do you think it's more easier for beginners init with ssh or https protocol?

Comment (mfrata) [reply template/README.md:30]: Https for sure. If i'm not mistaken, you have ve to have ssh setup to clone with git@...

Comment (ryukinix) [reply template/README.md:30]: Yes.

**Comment by vhdeluca** [comment]
Fully agree with all changes.

**Review by vhdeluca — APPROVED** [review]
Changes approved.

**Pull request merged**